tests/unit_tests/CMakeLists.txt

```diff
@@ -152,15 +152,15 @@ foreach(domain ${TARGET_DOMAINS})
   # Find individual tests within executable
     gtest_discover_tests(test_main_${domain}_rt
       PROPERTIES BUILD_RPATH ${CMAKE_BINARY_DIR}/lib
-      PROPERTIES ENVIRONMENT LD_LIBRARY_PATH=${CMAKE_BINARY_DIR}/lib:$ENV{LD_LIBRARY_PATH}
+      PROPERTIES ENVIRONMENT LD_LIBRARY_PATH=${CMAKE_BINARY_DIR}/lib:${CBLAS_LIB_DIR}:$ENV{LD_LIBRARY_PATH}
```

Comment:
I don't think we need this library for any other domains except BLAS, could you please add a check that `domain` is BLAS and only in this case add the extra library?

Reply:
Good point! Addressed in the commit [f762e6f](https://github.com/oneapi-src/oneMKL/pull/251/commits/f762e6f871af3eb77133374852d1b0ccf37f97ed).